test/integration/sign_test.go

```diff
@@ -38,11 +38,11 @@ const (
 
 type cleanupFn func() error
 
-func generateCosignKeyPair(t *testing.T, getPassFunc cosign.PassFunc) (privateKeyPath, publicKeyPath string, fn cleanupFn) {
```

Comment:
Looks like this is causing the test to throw NPE. If you want, you can drop this change and I can take care of it in a follow up.

Reply:
> Looks like this is causing the test to throw NPE. If you want, you can drop this change and I can take care of it in a follow up.

@xmudrii -- SGTM!